linux/system/init.sls

```diff
@@ -6,6 +6,9 @@ include:
 {%- if system.pkgs|length > 0 %}
 - linux.system.package
 {%- endif %}
+{%- if system.cpu_frequency is defined %}
+- linux.system.cpu_frequency
```

Comment:
I would name it only cpu to cover all cpu-related settings.
What you are touching is CPU frequency govenor.